src/CMakeLists.txt

```diff
@@ -133,11 +133,16 @@ execute_process(
 )
 string(REGEX REPLACE "[\r\n]" "" BUILD_DATE "${BUILD_DATE}")
 
-# Set the requirement for the C++ standard
-set(CMAKE_CXX_STANDARD 11)
```

Comment:
I'm not sure if I understand. We are not calling cmake from macports. This is called from TeamCity, which has a cmake command line on which now this is exposed.